packages/cli/__tests__/daemon/__integration__/disable/cli.daemon.disable.integration.ts

```diff
@@ -44,32 +43,26 @@ describe("daemon disable", () => {
         const sysInfo: ISystemInfo = ProcessUtils.getBasicSystemInfo();
 
         // form our tgz file name
-        let tgzFileName = "zowe-";
         switch (sysInfo.platform) {
             case "darwin": {
```

Comment:
Doesn't need to be implemented, but you could combine the case statements now. 😄

Reply:
Same response as above.